native-packages/fs-search/Cargo.toml

```diff
@@ -7,11 +7,11 @@ version = "0.1.0"
 crate-type = ["cdylib"]
 
 [dependencies]
-napi = { git = "https://github.com/napi-rs/napi-rs" }
-napi-derive = { git = "https://github.com/napi-rs/napi-rs" }
+napi = { git = "https://github.com/napi-rs/napi-rs", branch = "main" }
+napi-derive = { git = "https://github.com/napi-rs/napi-rs", branch = "main" }
 
 [build-dependencies]
-napi-build = { git = "https://github.com/napi-rs/napi-rs" }
+napi-build = { git = "https://github.com/napi-rs/napi-rs", branch = "main" }
```

Comment:
Somewhat unrelated, but can we just use the published versions like in the official template/docs? https://github.com/napi-rs/package-template/blob/main/Cargo.toml

Reply:
@devongovett was there a specific reason you were using the main branch? I imagine napi-rs has published since we were last pinned.